modules/support/client/api/block.api.js

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+
+export async function block(username) {
+  alert('alert2' + username);
+  return await axios.post(`/api/blocked-users/${username}`);
```

Comment:
Looks like the API route actually expects `put()` instead of `post()`:

https://github.com/Trustroots/trustroots/blob/48921b398eac5dcf2dd074b3b0afb5a3e6fa8416/modules/users/server/routes/users-block.server.routes.js#L14-L18